lib/utils/utils.go

```diff
@@ -682,6 +682,11 @@ func ByteCount(b int64) string {
 // ErrLimitReached means that the read limit is reached.
 var ErrLimitReached = &trace.LimitExceededError{Message: "the read limit is reached"}
 
+// ErrClientCredentialsHaveExpired means
+// that the credentials expired on the server-side,
+// and the user should relogin.
+var ErrClientCredentialsHaveExpired = &trace.AccessDeniedError{Message: "access denied: client credentials have expired, please relogin."}
```

Comment:
Could we find a more "thematic" place for this? Maybe api/client/errors.go, unless someone has a better idea?

Reply:
I found the `var ErrLimitReached` error which sits above, and there I added mine.
But that error is used only locally, in `utils.go` 🤦.

`api/client/errors.go` sounds good to me.